File: pset6/sentimental/mario.py
from cs50 import get_int

def main():
    print("Enter: ", end="")
    n = get_int()
    while n > 23 or n < 0:
        print("Enter: ", end="")
        n = get_int()
    for i in range(n):
        for q in range(n-i-1):
            print(" ", end="")

        for q in range(i+1):
            print("#", end="")
        print("  ", end="")
        for q in range(i+1):
            print("#", end="")
        print("")

if __name__ == "__main__":
    main()

File: pset1/mario.c
#include<stdio.h>
#include<cs50.h>
int main(void){
   int n = get_int("Height: ");
    while(n>23||n<0){
     n = get_int("Height: ");
    }
    for(int i = 0;i<n; i++){
        for(int  q=1;q<n-i;q++){
            printf(" ");
        }

        for(int q =-1;q<i;q++){
            printf("#");
        }
        printf("  ");

        for(int q =-1;q<i;q++){
            printf("#");
        }
        printf("\n");


    }

}